product_docs/docs/pgd/5/monitoring/sql.mdx

```diff
@@ -686,9 +686,9 @@ One of the following status summaries is returned:
 
 ## Monitoring transaction COMMITs
 
-By default, PGD transactions are only committed to the local node. In that case, a transaction's `COMMIT` is processed quickly.
+By default, PGD transactions are committed only to the local node. In that case, a transaction's `COMMIT` is processed quickly.
 
-PGD's commit scopes offer a range of [commit scopes](../durability/commit-scopes) with synchronous transaction options that allow you to balance durability, consistency, and performance for your particular queries. 
-You can monitor these transactions by examining the [`bdr.stat_activity`](/pgd/latest/reference/catalogs-visible#bdrstat_activity) catalog. The processes will report different `wait_event` states as a transaction is committed. This monitoring only covers transactions in progress and does not provide historical timing information.
+PGD offers a range of [commit scopes](../durability/commit-scopes) with synchronous transaction options that allow you to balance durability, consistency, and performance for your particular queries. 
```

Comment:
Commit Scopes (the feature) isn't just synchronous transaction commit scopes, it includes lag control which is not a synchronous transaction commit scope but a controlling commit scope.

PGD's [Commit Scopes](../durability/commit-scopes) feature offers a range of synchronous transaction commit scopes  that allow you to balance durability, consistency, and performance for your particular queries.